CHANGELOG.md

```diff
@@ -4,18 +4,18 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0]
 
-## [0.1.2] - Unreleased
-
-[0.1.2]: https://github.com/streem/streem-sdk-jvm/compare/v0.1.1...HEAD
+[0.2.0]: https://github.com/streem/streem-sdk-jvm/compare/v0.2.0...HEAD
 
```

Comment:
The version inside the compare link should've stayed 0.1.1 since it's meant to show what's changed since the last released version. Sorry I know the format here is a bit confusing. Anyways, I'll fix it up in the release commit.

Reply:
Bah, my bad